frontend/my-app/src/Pages/LandingPage.tsx

```diff
@@ -1,31 +1,31 @@
 import React, { useState } from 'react'
 import './LandingPage.css'
-import { useNavigate } from 'react-router-dom'
+import { useLocation, useNavigate } from 'react-router-dom'
+import type { User } from '../responseTypes'
 
 const LandingPage: React.FC = () => {
+  const location = useLocation()
+  console.log(location.state.friends)
```

Comment:
We should probably go through the codebase and annotate all the locations where we `console.log` with a comment like `// TODO: remove console.log`. Very bad practice to have those in production code, but for now I think as long as we mark them all we're okay

Reply:
Oop, will add those comments!